README.md

```diff
@@ -38,15 +47,43 @@ jobs:
+
+| Input                  | Description                                                                                                                                            | Default     |
+|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `calc`                 | Comma-separated list of [metrics](https://portal.tiobe.com/latest/docs/index.html#doc=user/clientoptions.html%23MetricAliases) to be calculated. The `GATE` metric is supported for TICS Viewer versions higher than 2022.2.x.              | `GATE`  |
+| `recalc`               | Comma-separated list of [metrics](https://portal.tiobe.com/latest/docs/index.html#doc=user/clientoptions.html%23MetricAliases) to be recalculated. The `GATE` GATE metric is supported for TICS Viewer versions higher than 2022.2.x. | -       |
+| `nocalc`               | Comma-separated list of [metrics](https://portal.tiobe.com/latest/docs/index.html#doc=user/clientoptions.html%23MetricAliases) to not be calculated.   | -           |
+| `norecalc`             | Comma-separated list of [metrics](https://portal.tiobe.com/latest/docs/index.html#doc=user/clientoptions.html%23MetricAliases) to not be recalculated. | -           |
+| `filelist`             | Path to a file containing the files (newline separated) to run TICS for. This can be an absolute or relative (to workspace) path, and can also be `.` to analyze the whole project. This has to be set when the action is run outside of a pull request. | -       |
+| `cdtoken`              | A custom client-data token for the purpose of the Client Viewer functionality. This provides a static URL that is updated with every analysis.              | -           |
+| `branchname`           | Name of the branch in TICS.                                                                                                                                 | -           |
+| `codetype`             | Allows you to pick which specific types of code you want to analyze with the TICS client. Options are `PRODUCTION`, `TESTCODE`, `EXTERNAL` and `GENERATED`. | `PRODUCTION`|
+| `excludeMovedFiles`    | Exclude moved and renamed files from analysis completely. By default these are included if there are modifications in the file.                             | `false`     |
+| `showBlockingAfter`    | Show the blocking after violations in the changed files window. Options are `true` or `false`.                                                              | `true`      |
+| `tmpdir`               | Location to store debug information.                                                                                                                        | -           |
+
+## QServer
+
+As of v3, the option to run [TICSQServer](https://ticsdocumentation.tiobe.com/latest/docs/#doc=admin/admin_A3_qserverref.html) analyses has been made available.

```

Comment:
We are pointing to two different documentation links here:

https://portal.tiobe.com/latest/docs/index.html#doc=user/clientoptions.html%23MetricAliases
https://ticsdocumentation.tiobe.com/latest/docs/#doc=admin/admin_A3_qserverref.html

We should use one or the other in the entire README.